config/simplesamlphp-users.php

```diff
@@ -13,42 +13,32 @@
     'example-userpass' => array(
         'exampleauth:UserPass',
         'user1:user1pass' => array(
-            'uid' => array('1'),
-            'eduPersonAffiliation' => array('group1'),
             /*
               NOTE: we add a bunch of claims that we expect to get back from Seneca,
               and need to simulate here.
             */
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress' => 'user1@myseneca.ca',
+            'displayname' => 'Johannes Kepler', 
```

Comment:
Is the display name the value we'll be displaying on the header?

Reply:
Currently we are showing sAMAccountName but we can and maybe should show display name instead, I don't know really.

Reply:
Also one more thing, what were the purpose of the schema links you removed?